### PR TITLE
Ukranian translation

### PR DESCRIPTION
**Pull request by Drommer**

### CODE REVIEW

File: locale/uk.po [hunk 1]
```diff
@@ -0,0 +1,916 @@
...
+#, python-brace-format
+msgid "AUR: Too many package results for '{query}'"
+msgstr "AUR: Дуже багато пакетів у результатах для '{query}'"
+
+#: pikaur/help_cli.py:44
+msgid ""
+"All common pacman options as when doing `pacman -U <pkg_file>`. See `pacman -"
+"Uh`."
+msgstr ""
+"Всі стандартні параметри pacman відповідають виконанню `pacman-U"
```
Comment (actionless) [inline locale/uk.po:76]: missing the space around command

File: locale/uk.po [hunk 2]
```diff
@@ -0,0 +1,916 @@
...
+msgstr "завжди перезбирати пакети з AUR"
+
+#: pikaur/help_cli.py:84
+msgid "always sysupgrade '-git', '-svn' and other dev packages"
+msgstr "завжди оновлювати пакети з '-git', '-svn' та інші нестабільні пакети"
+
+#: pikaur/install_cli.py:541 pikaur/install_cli.py:544 pikaur/build.py:707
+#: pikaur/build.py:717
+msgid "c"
+msgstr ""
```
Comment (actionless) [inline locale/uk.po:598]: it's empty

File: locale/uk.po [hunk 3]
```diff
@@ -0,0 +1,916 @@
...
+
+#: pikaur/help_cli.py:75
+msgid "cli args to pass to makepkg"
+msgstr "аргументи командного рядка для передачі makepkg"
+
+#: pikaur/build.py:707 pikaur/build.py:723
+msgid "d"
+msgstr ""
```
Comment (actionless) [inline locale/uk.po:606]: it's empty

**Comment by Drommer** [comment]
Corrected the error in line 76.

About blank lines. I did the same in the Russian locale. Everything appears correctly.

**Comment by actionless** [comment]
ok then, thanks a lot!

**Pull request merged**